index.html

```diff
@@ -31,17 +34,147 @@
+        <p class="profesional-art-p-2">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul-2">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-2">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
+      </div>
+      <div class="profesional-art-container-3">
+        <h2 class="profesional-art-h2-3">Profesional Art<br>Printing Data</h2>
+        <p class="profesional-art-p-3">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul-3">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-3">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
+      </div>
+      <div class="profesional-art-container-4">
+        <h2 class="profesional-art-h2-4">Profesional Art<br>Printing Data</h2>
+        <p class="profesional-art-p-4">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul-4">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-4">
+          <a href="index.html"><img src="img/seeproject.png"></a>
+        </nav>
+      </div>
+      <div class="profesional-art-container-5">
+        <h2 class="profesional-art-h2-5">Profesional Art<br>Printing Data</h2>
+        <p class="profesional-art-p-5">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+        <ul class="profesional-art-ul-5">
+          <il><a href="index.html"><img src="img/Taghtml2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagbootstrap2.png"></a></il>
+          <il><a href="index.html"><img src="img/Tagruby2.png"></a></il>
+        </ul> 
+        <nav class="profesional-art-nav-5">
+          <a href="index.html"><img src="img/seeproject.png"></a>
         </nav>
       </div>
     </section>
+    <section class="main-about-me-section">
+      <div class="about-me-container">
+        <h1 class="about-me-h1">About me</h1>
```

Comment:
- [ ] Please I suggest changing the `h1` tag to `h2-h6` tag to implement HTML semantics tags. This is a provided [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for more information. 😄